ksql-version-metrics-client/src/main/java/io/confluent/ksql/version/metrics/VersionCheckerAgent.java

```diff
@@ -23,4 +23,6 @@
 
   void start(KsqlModuleType moduleType, Properties ksqlProperties);
 
+  ActiveChecker getActiveChecker();
```

Comment:
No need to expose a type here. All that is needed is a method that can be called to say someone has made a request:

```suggestion
  void updateLastRequestTime();
```